packages/shared/components/atoms/tiles/tileContents/AliasActivityTileContent.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="typescript">
+    import { localize } from '@core/i18n'
+    import { InclusionState, IPersistedAsset, IAliasActivityData } from '@core/wallet'
+    import { truncateString } from '@lib/helpers'
+    import { Text, AssetIcon } from 'shared/components'
+    import { FontWeight } from 'shared/components/Text.svelte'
+
+    export let inclusionState: InclusionState
+    export let data: IAliasActivityData
+    export let asset: IPersistedAsset
+</script>
+
+<info-container class="flex flex-row items-center text-left space-x-4">
```

Comment:
You have this container and spacing across all contents should this be in the parent?